examples/custom-admin-ui-pages/admin/pages/custom-page.tsx

```diff
@@ -0,0 +1,34 @@
+/** @jsx jsx */
+import { Fragment } from 'react';
+import { jsx } from '@keystone-ui/core';
+// Please note that while this capability is driven by NextJS's pages directory
+// We do not currently expose any of the auxillary methods that NextJS provides i.e. `getStaticProps`
+// Presently the only export from the directory that is respected is the page component itself.
+export default function CustomPage() {
+  return (
+    <Fragment>
```

Comment:
This USED to leverage other components from @keystone-ui/core (Stack, Text etc.) but I redacted this, because I'm not comfortable using too much undocumented stuff in our examples :|

Reply:
@gwyneplaine I think it would make sense to show how to use it within the keystone admin layout (eg with header, nav, etc).

Seems like almost all use cases would want the page to render that way...

Reply:
@ChuckJonas  yep, that's why this was stalled. 
I'm going to merge this in in its current state for now, and add those changes shortly after once the nav customisation docs are out 👍